docs/operator-manual/applicationset/Template.md

```diff
@@ -108,3 +108,60 @@ spec:
 (*The full example can be found [here](https://github.com/argoproj/argo-cd/tree/master/applicationset/examples/template-override).*)
 
 In this example, the ApplicationSet controller will generate an `Application` resource using the `path` generated by the List generator, rather than the `path` value defined in `.spec.template`.
+
+## Patch Template
+
+Templating is only available on string type. However some uses cases may require to apply templating on other types.
+
+Example:
+
+- Set the automated sync policy
+- Switch prune boolean to true
+- Add multiple helm value files
+
+ArgoCD has a `patchTemplate` feature to allow advanced templating. It supports both json and yaml
```

Comment:
```suggestion
ArgoCD has a `templatePatch` feature to allow advanced templating. It supports both json and yaml
```

I beat my head against the wall for a surprisingly long time trying to figure out what I was doing wrong. 🤣

Reply:
Sorry about that. I'll correct it 😅

Reply:
Done